src/helpers/time.ts

```diff
@@ -14,9 +18,94 @@ export function formatCentiseconds(centiTime: number): string {
   return `${s}.${prefix(cs)}`;
 }
 
+/**
+ * Pads out a number with a 0 if it's under 10
+ * @param n
+ */
 function prefix(n: number): string {
   if (n < 10) {
     return `0${n}`;
   }
   return `${n}`;
 }
+
+export const SECOND_IN_CS = 100;
+export const MINUTE_IN_CS = 60 * SECOND_IN_CS;
+export const HOUR_IN_CS = 60 * MINUTE_IN_CS;
+
+interface PluralizeParams {
+  count: number;
+  word: string;
+  options?: {
+    fixed?: number;
+    abbreviate?: boolean;
+  };
+}
+
+/**
+ * Adds an s to a word if count is over 1.
+ * Takes options to pad the count and abbreviate the word with its
+ * first letter
+ * @param count
+ * @param word
+ * @param options
+ */
+export function pluralize({ count, word, options = {} }: PluralizeParams) {
```

Comment:
I wouldn't export this function, there's libraries out there that specialize in this use-case.

Reply:
I just exported it for testing. How do we do that?